assistant_stream.go

```diff
@@ -0,0 +1,82 @@
+package openai
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+var (
+	ErrStreamEventEmptyTopic    = errors.New("stream event gets an empty topic")
+	ErrStreamEventCallbackPanic = errors.New("occurs a runtime panic during the stream event callback")
+)
+
+type AssistantThreadRunStreamResponse struct {
+	ID     string       `json:"id"`
+	Object string       `json:"object"`
+	Delta  MessageDelta `json:"delta,omitempty"`
+}
```

Comment:
This is a very trimmed down version of the response model.

Reply:
The complete response has a lot many parameters which would be important to run function calling etc. https://platform.openai.com/docs/api-reference/runs/createRun